inst/schema/ValidateInputResponse.schema.json

```diff
@@ -1,11 +1,13 @@
 {
-	"id": "File",
+	"id": "ValidateInputResponse",
```

Comment:
It's up to you, but you can leave these `id`s off as they will be added for you as the schema is built